rotkehlchen/api/v1/schemas.py

```diff
@@ -2377,6 +2377,11 @@ def make_assets_post_query(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> dict[str, Any]:
+        try:  # convert evm addresses to checksum format, but also allow non-evm addresses.
+            address = to_checksum_address(data['address'])
+        except (ValueError, TypeError):
+            address = data['address']
```

Comment:
this way you avoid executing the logic for none

```suggestion
        if (address := data['address']) not None:
            # convert evm addresses to checksum format
            with supress(ValueError, TypeError):
                  address = to_checksum_address(data['address'])
```